pkg/driver/sqlite/sqlite_test.go

```diff
@@ -405,3 +405,18 @@ func TestSQLiteQuotedMigrationsTableName(t *testing.T) {
 		require.Equal(t, `"fooMigrations"`, name)
 	})
 }
+
+func TestSQLiteFTS5MigrationsTable(t *testing.T) {
+	drv := testSQLiteDriver(t)
+	drv.migrationsTableName = "test_migrations"
+
+	// prepare database
+	db := prepTestSQLiteDB(t)
+	defer dbutil.MustClose(db)
+	err := drv.CreateMigrationsTable(db)
+	require.NoError(t, err)
```

Comment:
There's no need to repeat testing `drv.CreateMigrationsTable()` in this test, as it's already covered by other tests.

This should be sufficient to replace L410-L417:

```
		drv := testSQLiteDriver(t)
		db := prepTestSQLiteDB(t)
		defer dbutil.MustClose(db)
```

Reply:
Based on reading `prepTestSQLiteDB`, I think that `drv := testSqliteDriver(t)` isn't needed here? Either way, understood, simplified, agreed.